client/src/components/Header/Header.js

```diff
@@ -1,92 +1,64 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Menu, MenuItem } from '@mui/material';
+import { TreeLogo } from '@/components/Icons';
+import HeaderLogo from '@/assets/images/logos/header-logo.svg';
 import { useAuth0 } from '@auth0/auth0-react';
 import { AuthButton } from '@/components/Auth';
-import HeaderLogo from '@/assets/images/logos/header-logo.svg';
+import Menu from './Menu';
+
 import './Header.scss';
 
 const Header = () => {
-  const [anchorEl, setAnchorEl] = React.useState(null);
-
-  const handleClick = (event) => {
-    setAnchorEl(event.currentTarget);
-  };
-
-  const handleClose = () => {
-    setAnchorEl(null);
-  };
   const { isAuthenticated } = useAuth0();
 
   return (
-    <div id="header" className="header">
-      <div className="header-content">
-        <Link to="/">
-          <img className="header--logo" src={HeaderLogo} alt="Header Logo" />
-        </Link>
-
-        <button
-          type="button"
-          className="header-btn-menu"
-          aria-controls="wtt-menu"
-          aria-haspopup="true"
-          onClick={handleClick}
-        >
-          &#9776;
-        </button>
-
-        <Menu
-          id="wtt-menu"
-          anchorEl={anchorEl}
-          keepMounted
-          open={Boolean(anchorEl)}
-          onClose={handleClose}
-        >
-          {isAuthenticated && (
-            <MenuItem onClick={handleClose}>
-              <Link to="/userprofile" className="header-link">
-                <HeaderButton menuItem="User Profile" />
-              </Link>
-            </MenuItem>
-          )}
-
-          <MenuItem onClick={handleClose}>
-            <Link to="/" className="header-link">
-              <HeaderButton menuItem="Map" />
+    <div className="header">
+      <div className="header__navigation">
```

Comment:
![image](https://user-images.githubusercontent.com/91238232/213937271-879bad55-9c43-46fb-b83e-818db650b82a.png)
I actually cant because the .header is centering the &__navigation div with different widths, I could use margins instead if it would make the nesting and redundancy a bit better.

Reply:
Oh, I see. 👍